.drone.yml

```diff
@@ -1,19 +1,35 @@
 ---
 kind: pipeline
 type: docker
-name: linux-amd64
+name: pcgc
 
 platform:
   arch: amd64
   os: linux
 
 steps:
+  - name: test
+    image: golang:1.13
+    volumes:
+      - name: deps
+        path: /go
+    commands:
+      - make test
+  - name: lint
+    image: golangci/golangci-lint:v1.21.0
+    commands:
+      - make lint
```

Comment:
does this not need the volume?

Reply:
no as it doesn't install anything, the `golangci-lint` comes with its own dependencies already build and just runs against the code